application/controllers/UserRoleController.php

```diff
@@ -250,8 +250,6 @@ public function actionRunExport($ptid)
      */
     public function actionDelete()
     {
-        $this->requirePostRequest();
```

Comment:
Disabling CRSF control is a security issue.

Reply:
Yes, agree, this is not OK.

Reply:
Oh yes : https://github.com/LimeSurvey/LimeSurvey/blob/62d4b30c3ec38c478551c6e35054e8a85cf38ce0/application/controllers/UserRoleController.php#L257

Right , we have a getPost here :)

resolved